liquidity-rewards/contracts/dependencies/PoolTokenWrapper.sol

```diff
@@ -12,25 +12,50 @@ contract PoolTokenWrapper {
     using SafeMath for uint256;
     using SafeERC20 for IERC20;
 
+    /**
+     * Pool token ERC20 token interface.
+     */
     IERC20 public poolToken;
 
+    /**
+     * Keep track total supply of the pool token deposited into this contract, this
+     * will reduce amount of calculations vs calling a balanceOf method.
+     */
```

Comment:
```suggestion
    /**
     * Keeps track of the total supply of the pool token deposited into this contract.
     * This will reduce the amount of calculations vs calling a balanceOf method.
     */
```